src/isolate-duplicates/index.js

```diff
@@ -1,3 +1,60 @@
-function isolateDuplicates(text) {}
+function isolateDuplicates(text) {
+
+  let result = [];
+  let consecutiveCharIndex = 1;
+  let numberDuplicate = 0;
+  let duplicateStr = '';
+
+  if(text === null || typeof text !== 'string'){
+    throw('Please enter a valid string');
+  }
+  else{
```

Comment:
You don't need this `else` statement as the program would exit, if the `if` condition is matched.